### PR TITLE
Add possibility to add favicon

### PR DESCRIPTION
**Pull request by albinahlback**
Edit: Saw now that it tried to search for `img/icon.png`. However, one should neither be restricted to use `png` format nor calling it `icon.png`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ paginate = 4
 [params]
   author = "Hugo de Groot"
   siteHeading = "Hugo Grotius" # defaults to author
+  # favicon = "favicon-gnu.png" # add favorite/URL/tab/shortcut icon
```
Comment (gevhaz) [inline README.md:52]:
```suggestion
  # favicon = "favicon-gnu.png" # Adds a small icon next to the page title in a tab
```

I guess your explanation is a more encompassing, but I'd like it a bit more descriptive for those who don't know what a favicon is. This should also be changed in the config.toml in that case.

**Review by gevhaz — COMMENTED** [review]
Hi Albin, thanks for the PR. A favicon is a nice addition :)

Have you checked if FSF has a licence for the Gnu icon/logo? Perhaps we need to credit them.

Otherwise, everything seems to work fine when I test locally. Just added a small suggestion.

**Comment by albinahlback** [comment]
> Hi Albin, thanks for the PR. A favicon is a nice addition :)
> 
> Have you checked if FSF has a licence for the Gnu icon/logo? Perhaps we need to credit them.
> 
> Otherwise, everything seems to work fine when I test locally. Just added a small suggestion.

I think I their terms wrong ([link](https://www.gnu.org/graphics/agnuhead.html)). One has to talk about them in a supportive manner in order to use it. And as GNU is not mentioned (except in the license), I think it should be removed. Do you have any images that can be used instead?

**Comment by gevhaz** [comment]
How about this one? https://commons.wikimedia.org/wiki/File:Hugo_Grotius00.jpg
Pros are that its public domain and it's Hugo Grotius so it goes with the theme of the example site. But you can't really see much detail in practice. Looks like this for me:

image

**Comment by albinahlback** [comment]
Yeah, that looks good, and is fitting for the example site!

**Comment by gevhaz** [comment]
Great :) I suggest picking the smallest variant.

**Comment by albinahlback** [comment]
I scaled it to 64x64 pixels, as, if I remember correctly, not all web-browsers support arbitrary format of favicons.

**Comment by albinahlback** [comment]
I think it should be ready now.

**Comment by gevhaz** [comment]
Good call with 64x64. 

Yes, looks good. I'll merge

**Comment by albinahlback** [comment]
And thank you for this theme, it is very nice! :-)

**Pull request merged**